src/meta/processors/partsMan/DropSpaceProcessor.cpp

```diff
@@ -43,7 +43,19 @@ void DropSpaceProcessor::process(const cpp2::DropSpaceReq& req) {
     deleteKeys.emplace_back(MetaServiceUtils::spaceKey(spaceId));
 
     // delete related role data.
-    // TODO(boshengchen) delete related role data under the space
+    auto rolePrefix = MetaServiceUtils::roleSpacePrefix(spaceId);
```

Comment:
Add some LOGs when deleting the users information.

Reply:
> Add some LOGs when deleting the users information.

fixed.